### PR TITLE
Improvements to remote debugging instructions

### PR DESCRIPTION
**Pull request by reshmabidikar**

### CODE REVIEW

File: userguide/tutorials/debugging.adoc [hunk 1]
```diff
@@ -52,13 +52,90 @@ curl -v http://<IP_ADDRESS>:8080/1.0/healthcheck?pretty=true
...
-If Kill Bill is running locally, you can just specify `localhost` or `127.0.0.1` as the remote host in your debugger. To debug instances remotely (e.g. in the cloud), you need to specify the publicly accessible IP address of your instance. For Docker, this should be the public IP address of the container. If you are using docker-machine and your machine is named `default`, you can retrieve the IP address via `docker-machine inspect default -f '{{.Driver.IPAddress}}'`. Passing this IP address to our Docker images as `ENV_HOST_IP` enables remote debugging automatically.
+In order to remote debug Kill Bill running in Tomcat, the following needs to be done:
 
-Make sure also that the port `12345` is open (e.g. check your security groups in AWS). For Docker, expose the port via `-p12345:12345`.
+1. Ensure that you have Kill Bill and Kaui installed in Tomcat as explained in the https://docs.killbill.io/latest/getting_started.html#_tomcat[Getting Started] guide.
+2. Set the following environment variable:  
++
+[source,bash]
+----
+JPDA_OPTS=-agentlib:jdwp=transport=dt_socket,address=8000,server=y,suspend=n
```
Comment (pierre) [inline userguide/tutorials/debugging.adoc]: For consistency with the instructions below, maybe switch to port `12345`?

File: userguide/tutorials/debugging.adoc [hunk 2]
```diff
@@ -52,13 +52,90 @@ curl -v http://<IP_ADDRESS>:8080/1.0/healthcheck?pretty=true
...
+----
++
+3. Start Tomcat as follows:
++
+[source,bash]
+----
+catalina.bat jpda start
```
Comment (pierre) [inline userguide/tutorials/debugging.adoc]: I suspect other docs use the Linux/MacOS instructions by default? If so, `catalina.sh`.

**Review by pierre — APPROVED** [review]

File: userguide/tutorials/debugging.adoc (hunk 1 above)
Comment (reshmabidikar) [reply userguide/tutorials/debugging.adoc]: Fixed

File: userguide/tutorials/debugging.adoc (hunk 2 above)
Comment (reshmabidikar) [reply userguide/tutorials/debugging.adoc]: Fixed

**Review by sbrossie — APPROVED** [review]

**Pull request merged**